package.json

```diff
@@ -87,5 +78,12 @@
       "soljson.js"
     ],
     "parser": "babel-eslint"
+  },
+  "dependencies": {
```

Comment:
oh sry. i thought modules required in a js file are supposed to be `dependencies` and tooling used during development (e.g. http-server, browserify, ...) belongs to dev-dependencies.